vue/src/app/playground/index.vue

```diff
@@ -1,11 +1,429 @@
 <template>
-  <div class="row">
-    <div class="example-spacing">
-      <!-- Insert Example Here -->
+  <div class="wrap">
```

Comment:
shouldn't this be left empty?  like we do in the other projects?  and these examples should be in example files

Reply:
Removed. This is the demo for engineers.